content/Watsonx/WatsonxAI/101.md

```diff
@@ -0,0 +1,352 @@
+
+With great artificial intelligence comes higher security risks. Solutions like ChatGPT are Very Large language Models (VLLMs) with 175 billion parameters. They fine-tuned by the OpenAI team using an additional non-public Chat datasets plus a Reinforcement Learning Human Feedback (RLHF) dataset. ChatGPT is a chatbot-enabled LLM.
+
+In watsonx.ai, we are interacting directly with smaller LLMs (3-20 billion parameters). This is a wise choice from a security POV. Prompt injection is a major risk for enterprise uses of LLMs. In prompt injections, a hacker will create an intricate prompt in order to cause a LLM such as ChatGPT to ignore/bypass security protocols and reveal sensitive company information. Just imagine you're a hacker. Which model would you choose to target for prompt injection hacking? OpenAI's ChatGPT with 175 billion parameters capable of thousands of tasks or a smaller, more-focused 3 billion parameter model highly tuned for a few isolated tasks? Which has a larger attack surface for prompt re-engineering?
+
+The smaller, simpler models in watsonx.ai are more challenging for potential hackers. Using many small models rather than a single large one such as ChatGPT creates a wider distribution of sensitive entry points. Each small language model is much harder to manipulate due their limited functionality and high level of prompt engineering that was required to perform their primary tasks. They don’t have the wide range of functions such as ChatGPT. As programmers know, putting all your resources into a single point of failure is unwise. It's far better to decompose your solution for security, scalability and control.
+
+That's right. For security, smaller is better. On top of security benefits, there are computational improvements with using smaller, lighter weight models as well. However we're getting ahead of ourselves. Let's first interact more with watsonx.ai's LLMs to better understand and learn how to make them respond the way we need.
+
+<br></br><br></br><br></br>
+
+### 5.0 Prompt engineering exercises
+
+Complete the following exercies using Watsonx.ai's Prompt Builder.
+
+<p>&nbsp;</p>
+
+### 5.1 Generate
+
+**Goal**
+
+<table>
+<tr>
+<td>
+Write three sentences about donkeys
+</td>
+</tr>
+</table>
+  
+**Samples**
+```
+3 sentences about puppies:
+- The puppy spun in circles, trying to catch his tail, but ended up tumbling over and over.
+- His antics had his owners laughing out loud, and even the other puppies at the park stopped to watch the silly sight.
+- As soon as the two puppies met at the park, their tails began to wag and they bounded around each other with glee.
+```
+```
+3 sentences about kittens:
+- The little kitten lapped up the milk with her tiny pink tongue, making a cute slurping sound.
+- The kitten nibbled on the treat, savoring every morsel of the delicious flavor.
+- As soon as the package was opened, the little kitten's eyes lit up with excitement.
+```
+
+See: [Example answer](109.md#1-generate)
```

Comment:
I am having issues going to the example answer page.
I am able to render the page if I include the full path `/watsonx/watsonxai/109#1-generate` like this